nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-core/src/main/java/org/nuxeo/ecm/platform/tag/BridgeTagService.java

```diff
@@ -51,6 +51,11 @@ public boolean hasFeature(Feature feature) {
         }
     }
 
+    @Override
+    public boolean supportTag(CoreSession session, String docId) {
+        return true;
+    }
```

Comment:
I think it's better here to check if both `first` and `second` tag service support tag.

Reply:
I will do an `OR` as we don't want to block some features during the bridge.